lib/archethic.ex

```diff
@@ -176,11 +205,23 @@ defmodule Archethic do
     TransactionChain.fetch_inputs_remotely(address, nodes, DateTime.utc_now())
   end
 
+  @spec available_locally?(binary) :: {boolean(), binary | nil}
+  def available_locally?(address) when is_binary(address) do
+    case get_last_transaction_address_locally(address) do
+      {:ok, last_address} -> {true, last_address}
+      {:error, :not_found} -> {false, nil}
```

Comment:
You should also catch `{:error, :network_issue}`
Also in case of wrong result, you can juste return false, nil is not used after

Reply:
right